### PR TITLE
DOCSUP-14783: Document the MySQL settings

### PR DESCRIPTION
**Pull request by sevirov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation for #24146

### CODE REVIEW

File: docs/ru/engines/table-engines/integrations/mysql.md [hunk 1]
```diff
@@ -102,8 +108,43 @@ SELECT * FROM mysql_table
...
+
+Задает размер пула соединений (если используются все соединения, запрос будет ждать, пока какое-либо соединение не будет освобождено).
+
+Возможные значения:
+
+-   Положительное целое число.
+
+Значение по умолчанию: `16`.
+
+## См. также {#smotrite-takzhe}
```
Comment (gyuton) [inline docs/ru/engines/table-engines/integrations/mysql.md:147]:
```suggestion
## См. также {#see-also}
```

**Comment by sevirov** [comment]
@kitaisreal Please, review this PR.

File: docs/en/engines/table-engines/integrations/mysql.md [hunk 2]
```diff
@@ -108,9 +108,43 @@ SELECT * FROM mysql_table
...
+Default settings are not very efficient, since they do not even reuse connections. These settings allow you to increase the number of queries run by the server per second.
+
+### connection_auto_close {#connection-auto-close}
+
+Allows to automatically close the connection after query execution, i.e. disable connection reuse.
+
+Possible values:
+
+-   1 — The connection reuse is disabled.
+-   0 — The connection reuse is enabled.
```
Comment (vdimir) [inline docs/en/engines/table-engines/integrations/mysql.md:122]:
```suggestion
-   1 — Auto-close connection is allowed, so the connection reuse is disabled
-   0 — Auto-close connection is not allowed, so the connection reuse is enabled
```

File: docs/ru/engines/table-engines/integrations/mysql.md [hunk 3]
```diff
@@ -102,8 +108,43 @@ SELECT * FROM mysql_table
...
+Настройки по умолчанию не очень эффективны, так как они не используют повторное соединение. Эти настройки позволяют увеличить количество запросов, выполняемых сервером в секунду.
+
+### connection_auto_close {#connection-auto-close}
+
+Позволяет автоматически закрыть соединение после выполнения запроса, то есть отключить повторное использование соединения.
+
+Возможные значения:
+
+-   1 — повторное использование соединения отключено.
+-   0 — включено повторное использование соединения.
```
Comment (vdimir) [inline docs/ru/engines/table-engines/integrations/mysql.md:122]:
```suggestion
-   1 — автоматическое закрытие соединения разрешено (повторное использование отключается).
-   0 — автоматическое закрытие соединения запрещено (повторное использование включается).
```

**Review by vdimir — APPROVED** [review]

**Pull request merged**